indico/htdocs/sass/partials/_core.scss

```diff
@@ -104,3 +104,7 @@
     overflow-x: hidden;
     overflow-y: auto;
 }
+
+.truncate-text {
```

Comment:
I would still stick with `truncate`.
https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/
What other options are there for truncating a text? You would never truncate it without using `...` at the end, it would look ugly and in any case we would never use a different class to implement this behavior.